packages/panzer/adapters-stk/example/PoissonExampleCurvilinear/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
If we don't want to check the L2/H1 error, this can be easily incorporated into main_driver.
A better short term solution may be to modify PoissonExample to always do the area calculation, allow for a mesh to be supplied, but grab different reference solutions depending on the test case.
These examples probably require a bit of an overhaul at some point. We should discuss.